### DISCUSSION TITLE
is there a way to map M3 to mute the mic?

### DISCUSSION THREAD

**Discussion by iakeehc** [seerge/g-helper#141] [Q&A] [answered]
Hi, I would like to map the M3 key to mute the mic as it did previously, is there anyway to do so?

**Comment by seerge** [comment, marked as answer]
Currently - no, cause for me asus optimization service seem to do the muting (so app deliberately doesn't interfere with that)?

**Reply by iakeehc** [reply]
ok thanks!

**Comment by sklynic** [comment]
When asus optimization work the m3 (mute mic) actually work (you can check in windows settings by watching the mic volume bar). However there's no notification like mute volume if other asus services are disabled so it's comfusing
